editor/src/components/canvas/ui-jsx-canvas-errors.spec.tsx

```diff
@@ -684,11 +684,117 @@ export var ${BakedInStoryboardVariableName} = (props) => {
     expect(canvasErrors).toMatchInlineSnapshot(`
       Array [
         Object {
-          "columnNumber": undefined,
-          "lineNumber": undefined,
-          "message": "Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.",
+          "columnNumber": 32,
+          "lineNumber": 5,
+          "message": "(MyCard) Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined.",
           "name": "Error",
-          "originalCode": undefined,
+          "originalCode": Array [
+            ScriptLine {
+              "content": "import * as React from \\"react\\"",
+              "highlight": false,
+              "lineNumber": 1,
+            },
+            ScriptLine {
+              "content": "      import { View, Storyboard, Scene } from 'utopia-api'",
+              "highlight": false,
+              "lineNumber": 2,
+            },
+            ScriptLine {
+              "content": "",
+              "highlight": false,
+              "lineNumber": 3,
+            },
+            ScriptLine {
+              "content": "      const MyCard = undefined",
+              "highlight": false,
+              "lineNumber": 4,
+            },
+            ScriptLine {
+              "content": "      export var App = props => <MyCard data-uid={'bbb'} />",
+              "highlight": false,
```

Comment:
I don't remember if we need this to be `highlight: true` or not – it might be that the error overlay is not reading this and using the line number from above, but I can also imagine that this is a flag that needs to be set in order to highlight this line

Reply:
<img width="1102" alt="Screenshot 2021-06-02 at 10 07 03" src="https://user-images.githubusercontent.com/4403069/120446934-54c48300-c38a-11eb-9ea2-72d4f0c098fb.png">

Reply:
looks great :)